cts/environment.py

```diff
@@ -88,11 +87,6 @@ def SeedRandom(self, seed=None):
         if not seed:
             seed = int(time.time())
 
-        if self.has_key("RandSeed"):
-            self.logger.log("New random seed is: " + str(seed))
-        else:
-            self.logger.log("Random seed is: " + str(seed))
-
```

Comment:
It's printed again immediately after this with the rest of the settings, so this is redundant.